docs/source/roadmap.rst

```diff
@@ -16,15 +16,11 @@ contributions. Here's our plans for the next two releases.
     the feature. Pull Requests are open to anyone.
 
 
-3.3
+3.4
 ---
-
-* **RHEL/CentOS:** Drop support for RHEL/CentOS 6.x.
-* **Mistral deprecation:** Orquesta replaces Mistral as the workflow engine.
-* **HipChat Removal:** The HipChat adapter has been removed from st2chatops.
-* **Chef:** Deprecated `chef-stackstorm <https://github.com/stackstorm/chef-stackstorm>`_ deployment.
-* **Docker:** Overhaul for `st2-docker <https://github.com/stackstorm/st2-docker>`_ deployment.
-
+* **Python:** Drop Python 2 support, rely on Python 3 ony.
```

Comment:
Interesting on py 2. I don't think we can say for sure we'll have it all removed in the 3.4 release (plan to be decided tomorrow) - especially as still debate on U16 to be had. I would think it's safe to say we will be starting on it - but if we are going for fixed 3 month releases - have we agreed exactly what is going to be in 3.4.
It would be safe to say its for upcoming release, but just wondering if we want to commit that it is 3.4.

Have we agreed what is in 3.4? Or should all of these be in the planned future but not scheduled yet?

Reply:
Fixed